### ISSUE TITLE
Add EBS pricing

### ISSUE THREAD

**Issue by ifosch** [ifosch/accloudtant#98]
I want to get pricing for EBS

**Comment by wtip** [comment]
Is EBS pricing currently not being accounted for? I saw this block https://github.com/ifosch/accloudtant/blob/master/accloudtant/aws/prices.py#L342 for ebs pricing but I couldn't tell if it's being using by anything at this point

**Comment by ifosch** [comment]
Nope, EBS pricing is not accounted. This was a draft on the first parsing code implemented.
I have a plan to discard this parsing code at all, with a newer implementation using the newer AWS Prices API, though. Right now, it is only providing accountancy on EC2 (non-spot) instances, and with some small bug about reservations.
